docs/advanced-topics/advanced-methods.md

```diff
@@ -0,0 +1,100 @@
+(advanced-methods)=
+# {octicon}`mortar-board` Advanced Methods
+Many of the models supported by finetuner make use of similar methods throughout finetuning, such as the methods used for *calculating loss*, *sampling* and *pooling* . Finetuner offers alternatives to each of these methods, which can improve the performance of your finetuning run in some cases.
+
+## Loss Functions
+
+Loss functions are used to calculate the quality of embeddings while training; the higher the output of the loss function, the more the parameters of the model will be updated.
+By default we use Triplet Margin Loss, however we support many other loss function as well, including ArcFace Loss and CosFace Loss.
```

Comment:
```suggestion
By default we use `'TripletMarginLoss'`, however we support many other loss function as well, including ArcFace Loss and CosFace Loss.
```

or withouth the quotes, but I would prefer to write it as it is used in the fit function